slider/beatmap.py

```diff
@@ -660,6 +668,30 @@ def tick_points(self):
             ),
         )
 
+        if self.lazy_slider_end:
+            # curve() takes in a percentage of how far along we want the point.
+            # Take away the offset from the total length of the slider to get
+            # the percentage of the slider we want the point at.
+            true_end_time = self.end_time
+
+            if self.has_updated_end or self.lazy_slider_end:
+                true_end_time -= timedelta(milliseconds=LAZY_SLIDER_END_OFFSET)
```

Comment:
as far as I can tell reading lazer code, the end time of sliders is not impacted by this legacy tick offset. I think lazer's direction is a good one to follow since the concepts of "last slidertick in a slider" and "visual end time of a slider" are distinct.

Reply:
Agreed, i somehow thought the end timing was also affected, when it isnt. Feel free to remove this, i will only come to it late friday ~18UTC+1